docs/interact/indexer.mdx

```diff
@@ -1,36 +1,76 @@
 ---
-title: Fetch data from the Indexer
+title: Indexer API
 sidebar_position: 4
 ---
 
-## Prerequisites
-Access to the Calimero console running a private shard.
+### Prerequisites
+You must have an access to the Calimero console to interact with Indexer.
 
-## Configuration
+### Configuration
 
-Create a new shard token
-https://near.github.io/near-api-js/interfaces/connect.connectconfig.html
+To be able to call Indexer API you must set few parameters:
+1. Endpoint
+2. Authorization token
 
-Generate a new token in the calimero security dashboard.
+#Endpoint
+Endpoint is constructed from multiple parts:
```

Comment:
This can be copied from the dashboard, we should mention it

Reply:
Added link to a subpage where it is explained. 
<img width="736" alt="Screenshot 2022-11-29 at 00 29 37" src="https://user-images.githubusercontent.com/12198572/204402363-a4d55e82-254d-4fed-9460-36727baa1b06.png">

Reply:
<img width="700" alt="Screenshot 2022-11-29 at 00 30 37" src="https://user-images.githubusercontent.com/12198572/204402489-bdb866a0-1138-40a5-85a8-1281d0b7662f.png">

added link to the subpage where it is explained

Reply:
Although looking at this section, seems like links are outdated. I will update them